### PR TITLE
improve top level docs

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

**Comment by nslater** [comment]
@chaudum @mfussenegger only one of you needs to ack. sorry. I tagged jordi first but realised maybe haudi would prefer to review ^^

File: README.rst [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
+  results from Microsoft Azure.
 
-api
-===
+Contributing
+============
 
-This folder contains the API (Flask application) to access benchmark results
-stored in the Crate production cluster on Microsoft Azure.
+This project is primarily maintained by `Crate.io`_, but we welcome community
```
Comment (chaudum) [inline README.rst]: this repo should not have a contributing section

File: api/README.rst [hunk 2]
```diff
@@ -1,86 +1,71 @@
...
-    runtime_stats['max'] as "max",
-    runtime_stats['stdev'] as "stdev",
-    runtime_stats['variance'] as "variance",
-    statement
-  FROM "benchmark"."history"
-  WHERE statement = ANY(?)
-    AND version_info['build_timestamp'] >= ?
-    AND version_info['build_timestamp'] <= ?
-  ORDER BY version, statement
+      SELECT @BENCHMARK_GROUP as "group",
```
Comment (chaudum) [inline api/README.rst]: `@BENCHMARK_GROUP` isn't a valid parameter substitution, is it?

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
+============
```
Comment (chaudum) [inline CONTRIBUTING.rst]: don't need that IMO

Comment (nslater) [inline CONTRIBUTING.rst]: the double "=" is needed to mark it as the doc title on GitHub

File: README.rst (hunk 1 above)
Comment (nslater) [inline README.rst]: okay! will remove!

File: api/README.rst (hunk 2 above)
Comment (nslater) [inline api/README.rst]: nope. but I wanted to make it clear to the reader which values get substituted with what. I originally wrote them as <BENCHMARK_GROUP> but then that messes up the SQL syntax highlighting. so I switched to using SQL variable syntax

I forgot to add a para explaining this though. would you be happy if I added one saying, "here, `$BENCHMARK_GROUP` is substituted for `<BENCHMARK_GROUP>` in the API call" or something similar?

File: CONTRIBUTING.rst (hunk 3 above)
Comment (nslater) [inline CONTRIBUTING.rst]: oh. this file is gone now anyway. haha

Comment (chaudum) [inline CONTRIBUTING.rst]: I meant the CONTRIBUTING file in general, because it is only maintained by us

Comment (nslater) [inline CONTRIBUTING.rst]: aah!

**Comment by nslater** [comment]
@chaudum okay I've addressed all your comments!!

**Pull request merged**